unittests/Basic/FileSystemTest.cpp

```diff
@@ -54,15 +54,15 @@ TEST(FileSystemTest, basic) {
   EXPECT_EQ(missingFileContents.get(), nullptr);
 
   auto ourFileContents = fs->getFileContents(tempPath.str());
-  EXPECT_EQ(ourFileContents->getBuffer().str(), "Hello, world!\n");
+  EXPECT_EQ(ourFileContents->getBuffer().str(), "Hello, world!");
 
   // Remote the temporary file.
   auto ec = llvm::sys::fs::remove(tempPath.str());
   EXPECT_FALSE(ec);
 }
 
 TEST(FileSystemTest, testRecursiveRemoval) {
-  TmpDir rootTempDir{ __FUNCTION__ };
+  TmpDir rootTempDir(__func__);
```

Comment:
`__FUNCTION__` in MSVC doesn't give the same thing as `__FUNCTION__` in Clang. `__func__` however is standardized in C++11 and is the same thing as `__FUNCTION__` with Clang